lib/src/base-styles/_nx-colors.scss

```diff
@@ -485,6 +490,11 @@
     --nx-color-policy-threat-slider-text-disabled: var(--nx-color-policy-threat-slider-border-disabled);
     --nx-color-policy-threat-slider-text: var(--nx-swatch-indigo-97);
 
+    // NxTable
+    --nx-color-table-header-background: var(--nx-swatch-indigo-35);
+    --nx-color-table-row-background-hover: var(--nx-swatch-indigo-15);
```

Comment:
This matches `NxList`, so there should probably be a common variable for at least those two.  I am however wondering why they use indigo-15 while other interactive elements (including other elements that have a `selected` state, like `NxCollapsibleItems` use indigo-10.

Reply:
>I am however wondering why they use indigo-15 while other interactive elements (including other elements that have a `selected` state, like `NxCollapsibleItems` use indigo-10.

I know this was brought up by Abhinav a week or two ago, and he had a meeting with Jason to confirm the interaction states and choices made: https://sonatype.slack.com/archives/CJUF9G7T2/p1681828099480709

Reply:
> This matches NxList, so there should probably be a common variable for at least those two. I am however wondering why they use indigo-15 while other interactive elements (including other elements that have a selected state, like NxCollapsibleItems use indigo-10.

Looking through the dark mode hover variables, `indigo-15` is used on hover for secondary/tertiary buttons, dropdownMenus, tabs, and tables. (`indigo-10` used by collapsibleItems, pagination & iconOnly buttons I believe). Would a valid generic name simply be `--nx-color-interactive-background-hover-secondary`?